src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_matmul_to_fc.cpp

```diff
@@ -147,9 +155,17 @@ ov::intel_cpu::ConvertMatMulToFC::ConvertMatMulToFC() {
             fc_input_a = create_transpose(fc_input_a, matmul->get_friendly_name() + "/transpose_a");
         }
 
-        auto output_rank = matmul->get_output_partial_shape(0).rank();
+        // Create new Convert
```

Comment:
It is not new convert. We just change connections

Reply:
fixed, thanks